src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/HenleggBehandling/HenleggBehandling.tsx

```diff
@@ -108,108 +105,115 @@ const HenleggBehandling: React.FC<IProps> = ({ fagsakId, behandling }) => {
             >
                 Henlegg behandling
             </Dropdown.Menu.List.Item>
-
-            <UIModalWrapper
-                modal={{
-                    actions: [
-                        <StyledLenke
-                            key={'forhåndsvis'}
-                            href="#"
-                            onClick={() => {
-                                hentForhåndsvisning({
-                                    method: 'POST',
-                                    data: hentSkjemaData(),
-                                    url: `/familie-ba-sak/api/dokument/forhaandsvis-brev/${behandlingId}`,
-                                });
-                            }}
-                            visLenke={skjema.felter.årsak.verdi === HenleggÅrsak.SØKNAD_TRUKKET}
-                        >
-                            Forhåndsvis
-                        </StyledLenke>,
-                        <Button
-                            key={'avbryt'}
-                            variant="tertiary"
-                            size="small"
-                            onClick={() => {
-                                nullstillSkjema();
-                                settVisModal(false);
-                            }}
-                            children={'Avbryt'}
-                        />,
-                        <Button
-                            key={'bekreft'}
-                            variant="primary"
-                            size="small"
-                            onClick={() => onBekreft(behandling.behandlingId)}
-                            loading={skjema.submitRessurs.status === RessursStatus.HENTER}
-                            disabled={skjema.submitRessurs.status === RessursStatus.HENTER}
-                            children={
-                                skjema.felter.årsak.verdi === HenleggÅrsak.SØKNAD_TRUKKET
-                                    ? 'Bekreft og send brev'
-                                    : 'Bekreft'
-                            }
-                        />,
-                    ],
-                    onClose: () => {
-                        nullstillSkjema();
-                        nullstillDokument();
-                        settVisModal(false);
-                    },
-                    lukkKnapp: true,
-                    tittel: 'Henlegg behandling',
-                    visModal,
+            <StyledModal
+                open={visModal && !visDokumentModal}
```

Comment:
Skjuler modalen hvis forhåndsvisningsmodalen vises, for å unngå at modalene ligger feil oppå hverandre. Har testet at dette gir riktig oppførsel og tilstand :)